galaxy/values.yaml

```diff
@@ -261,8 +261,8 @@ configs:
           {{- end }}
         k8s_namespace: "{{ .Release.Namespace }}"
         k8s_galaxy_instance_id: "{{ .Release.Name }}"
-        k8s_run_as_user_id: "101"
-        k8s_run_as_group_id: "101"
+        k8s_run_as_user_id: "0"
+        k8s_run_as_group_id: "0"
```

Comment:
Does this need to be run as root?

Reply:
No, removing them to nullify is better, as long as it doesn't affect other tools negatively